models/gantt.model.task.js

```diff
@@ -1,30 +1,30 @@
-import { Schema, model } from 'mongoose';
+import { Schema, model } from "mongoose";
```

Comment:
Did not change anything here. Just formatted the code. You have to see the model of project and modify this accordingly.